<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputMatrix</class>
 <widget class="QDialog" name="InputMatrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="MatrixName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_Btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>291</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>40</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="1" column="0">
     <widget class="QLineEdit" name="m4"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="m6"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="m5"/>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="m1"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="m3"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="m2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="m7"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="m8"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="m9"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
